specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -3532,23 +3532,20 @@
           "description": "Database name InstanceName/DataBaseName for SQL or System/DbName for SAP Hana",
           "type": "string"
         },
-        "targetDirectoryMapping": {
-          "description": "This will contain the target folder mapping for the Full/Diff/Log/Incremental pits.",
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          }
+        "targetDirectoryForFileRestore": {
```

Comment:
Is renaming 'targetDirectoryMapping' to 'targetDirectoryForFileRestore' a correction to your swagger or an API change?

Reply:
Hi Ryan. It is a correction to the swagger. It was already updated to targetDirectoryForFileRestore in the latest version 2019-06-15 and merged. The version 2018-12-20 was created before 2019-06-15 but it got merged sometime back. We need to publish the SDK for the new APIs added in 2018-12-20. So to avoid a breaking change we are adding this version in the latest package 2019-06. So in order to make this API consistent with the latest version I have made this update.